code/game/dna/genes/disabilities.dm

```diff
@@ -7,113 +7,143 @@
 /////////////////////
 
 /datum/dna/gene/disability
-	name="DISABILITY"
+	name = "DISABILITY"
 
 	// Mutation to give (or 0)
-	var/mutation=0
-
-	// Disability to give (or 0)
-	var/disability=0
+	var/mutation = 0
 
 	// Activation message
-	var/activation_message=""
+	var/activation_message = ""
 
 	// Yay, you're no longer growing 3 arms
-	var/deactivation_message=""
+	var/deactivation_message = ""
 
-/datum/dna/gene/disability/can_activate(var/mob/M,var/flags)
-	return 1 // Always set!
+/datum/dna/gene/disability/can_activate(mob/M, flags)
+	return TRUE // Always set!
 
-/datum/dna/gene/disability/activate(var/mob/living/M, var/connected, var/flags)
+/datum/dna/gene/disability/activate(mob/living/M, connected, flags)
 	..()
-	if(mutation && !(mutation in M.mutations))
-		M.mutations.Add(mutation)
-	if(disability)
-		M.disabilities|=disability
+	M.mutations |= mutation
```

Comment:
This is `|` because sometimes there's more than one source of stuff like mute/nearsighted/deaf. It's kinda dumb

That'll all go away and make 100% more sense once we have traits.